app/models/bars/BarsVerifyResponse.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.bars
+
+import models.bars.BarsCommon.BarsAssessmentType
+import play.api.libs.json.{Format, Json}
+
+final case class BarsVerifyResponse(
+    accountNumberIsWellFormatted:             BarsAssessmentType,
+    accountExists:                            BarsAssessmentType,
+    nameMatches:                              BarsAssessmentType,
+    nonStandardAccountDetailsRequiredForBacs: BarsAssessmentType,
+    sortCodeIsPresentOnEISCD:                 BarsAssessmentType,
+    sortCodeSupportsDirectDebit:              BarsAssessmentType,
+    sortCodeSupportsDirectCredit:             BarsAssessmentType,
+    accountName:                              Option[String],
```

Comment:
Maybe we could reuse the types from essttp backend where we have case classes for accountName etc instead of String, but feel free to ignore as this is supposed to be reused by oceans 11

Reply:
I'm avoiding using essttp types in shared code here.

Reply:
I've just realised I've let `essttp.rootmodel` creep into parts of BarsCommon. I need to fix that...